src/containers/Leagues/Leagues.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import "./Leagues.scss";
+import setLeagues from "../../actions/actionLeagues";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+import { STANDINGS, LEAGUES} from "../../some-staff/consts";
```

Comment:
я бы переместил куда-то :)
```suggestion
import { STANDINGS, LEAGUES} from "../../utils/consts";
```